doc/source/cd/methods/spotthediffdrift.ipynb

```diff
@@ -123,7 +123,7 @@
     "```python\n",
     "import tensorflow as tf\n",
     "from tensorflow.keras.layers import Conv2D, Flatten, Input\n",
-    "from alibi_detect.utils.tensorflow.kernels import DeepKernel\n",
+    "from alibi_detect.utils.tensorflow import DeepKernel\n",
```

Comment:
Change also needs to be made in main text: `The DeepKernel class found in either alibi_detect.utils.tensorflow.kernels or alibi_detect.utils.pytorch.kernels`

Might be worth grep-ing all the import statements that have been changed?

Reply:
Hmm, good idea. I found `alibi_detect.utils.pytorch.data.TorchDataset` and `alibi_detect.utils.tensorflow.data.TFDataset` as well, These are referenced in a parameter docstring. Are they intended to be public? Would a user import and use them? They're not used explicitly anywhere within the examples!

Reply:
In offline discussion agreed the `TorchDataset` and `TFDataset`'s should be public so I've protected them with `import_optional` and fixed private import paths in the docs.